### ISSUE TITLE
Automatically wrap headers with anchor tags

### ISSUE THREAD

**Issue by arowla** [18F/pages#40]
Can we use something like [Redcarpet](https://george-hawkins.github.io/basic-gfm-jekyll/redcarpet-extensions.html#with_toc_data) to automatically make headers in 18F pages linkable?

**Comment by mbland** [comment]
Maybe... Haven't played with it. Might be worth a shot. But, I'm not sure that it would be a built-in feature of Pages, but something we could add to the `guides_template_18f` gem or something.

**Comment by arowla** [comment]
Yeah, right after I posted I wondered if I had picked the wrong repo. :)

On Tue, Sep 22, 2015 at 3:34 PM, Mike Bland notifications@github.com
wrote:

> Maybe... Haven't played with it. Might be worth a shot. But, I'm not sure
> that it would be a built-in feature of Pages, but something we could add to
> the guides_template_18f gem or something.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/18F/pages/issues/40#issuecomment-142394001.

## 

Alison Rowland
Technical Lead, https://fbopen.gsa.gov | 18F
202-317-0124

**Comment by afeld** [comment]
I have done this in JS before:

https://github.com/afeld/hackerhours.org/blob/gh-pages/scripts/anchor.coffee

I know not all Markdown parsers automatically add `id`s for headings, but [Kramdown](http://kramdown.gettalong.org/index.html) does by default.
